test/recipes/80-test_cmp_http.t

```diff
@@ -278,19 +276,25 @@ sub start_mock_server {
     my $pid = open($server_fh, "$cmd|") or die "Trying to $cmd";
     print "Pid is: $pid\n";
     if ($server_port eq "0") {
+        my $accept_msg;
+
         # Find out the actual server port
         while (<$server_fh>) {
             print;
             s/\R$//;                # Better chomp
             next unless (/^ACCEPT/);
-            $server_port = $server_tls = $kur_port = $pbm_port = $1
-                if m/^ACCEPT\s.*?:(\d+)$/;
+            $accept_msg = $1 if /^ACCEPT\s.*:(\d+)$/;
```

Comment:
The name `accept_msg` is not (or: no more) to the point,
and in fact it can be replaced by simply using server_port, as it was before.

Reply:
Okie.  I pushed a fixup along tose lines